packages/ruby/README.md

```diff
@@ -58,13 +58,21 @@ options = {
 }
 
 config.middleware.use Readme::Metrics, options do |env|
-  current_user = env['warden'].authenticate(scope: :current_user)
+  current_user = env['warden'].authenticate
 
-  {
-    id: current_user.id
-    label: current_user.full_name,
-    email: current_user.email
-  }
+  if current_user.present?
+    {
+      id: current_user.id,
+      label: current_user.name,
+      email: current_user.email
+    }
+  else
+    {
+      id: "My Rails Application",
```

Comment:
Would it make sense to use something that looks like a real user id here? Something like `0` or `-1`, or even the string `"guest"`? What would you expect `Guest#id` to return?